components/signersForm.tsx

```diff
@@ -0,0 +1,295 @@
+import { validateAddress } from "@taquito/utils";
+import {
+    ErrorMessage,
+    Field,
+    FieldArray,
+    Form,
+    Formik,
+    FormikErrors,
+} from "formik";
+import { FC, useContext, useState } from "react";
+import { AppDispatchContext, AppStateContext } from "../context/state";
+import { content } from "../context/types";
+import ContractLoader from "./contractLoader";
+function get(
+    s: string | FormikErrors<{ name: string; address: string }>
+): boolean {
+    if (typeof s == "string") {
+        return false;
+    } else {
+        if (s.address) {
+            return s.address.length !== 0;
+        } else {
+            return false;
+        }
+    }
+}
+const SignersForm: FC<{ closeModal: () => void; address: string }> = (
+    props
+) => {
+    const state = useContext(AppStateContext)!;
+    let dispatch = useContext(AppDispatchContext)!
+
+    let [loading, setLoading] = useState(false);
+    let [result, setResult] = useState("");
+    if (state?.address == null) {
+        return null;
+    }
+
+    const renderError = (message: string) => {
+        return <p className="italic text-red-600">{message}</p>;
+    };
+    const initialProps: {
+        validators: { name: string; address: string }[];
+        requiredSignatures: number;
+    } = {
+        validators: state.contracts[props.address].signers.map((x) => ({
+            address: x,
+            name: state.aliases[x] || "",
+        })),
+        requiredSignatures: state.contracts[props.address].threshold,
+    };
+    async function changeSettings(
+        txs: { name: string; address: string }[],
+        requiredSignatures: number
+    ) {
+        let cc = await state.connection.contract.at(props.address);
+        let initialSigners = new Set(state.contracts[props.address].signers);
+        let input = new Set(txs.map((x) => x.address));
+        let removed = new Set(
+            [...initialSigners.values()].filter((x) => !input.has(x))
+        );
+        let added = new Set(
+            [...input.values()].filter((x) => !initialSigners.has(x))
+        )
+        let ops: content[] = []
+        if (added.size > 0) {
+            ops.push({ add_signers: [...added.values()] })
+        }
+        if (removed.size > 0) {
+            ops.push({ remove_signers: [...removed.values()] })
+        }
+        if (state.contracts[props.address].threshold === requiredSignatures) {
+            ops.push({ adjust_threshold: requiredSignatures })
+        }
+        let params = cc.methods
+            .create_proposal(ops)
```

Comment:
it seems something wrong here. the updated setting didn't show in params.  A warning from beacon shows "Transaction is likely to fail"

Reply:
it works when using new version wallet.